sdk/identity/azure-identity/pom.xml

```diff
@@ -98,6 +98,7 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Optional: This dependency can be excluded if IntelliJ Credential is not being used for authentication via `IntelliJCredential` or `DefaultAzureCredential` -->
```

Comment:
If a GA version of Identity is not shipped with `KeePassJava2` dependency in compile scope, we can set `<optional>true</optional>`